domain/consensus/utils/hashserialization/utxo.go

```diff
@@ -49,15 +49,18 @@ func serializeOutpoint(w io.Writer, outpoint *externalapi.DomainOutpoint) error
 }
 
 func serializeUTXOEntry(w io.Writer, entry *externalapi.UTXOEntry) error {
-	buf := [8 + 1 + 8]byte{}
+	buf := [8 + 8]byte{}
```

Comment:
I'm not sure what's the point of this. Please just call WriteElement for BlockBlueScore and Amount.